test/Distributed/Runtime/distributed_actor_whenLocal.swift

```diff
@@ -30,18 +30,20 @@ func test() async throws {
 
   let local = Capybara(actorSystem: system)
   // await local.eat() // SHOULD ERROR
-  let valueWhenLocal: String? = await local.whenLocal { __secretlyKnownToBeLocal in
-    __secretlyKnownToBeLocal.eat()
+  let valueWhenLocal: String? = await local.whenLocal {
+    (definitelyLocal: _local Capybara) in await definitelyLocal.eat()
```

Comment:
so explicitly annotating works, but it doesnt propagate implicitly if we just say `definitelyLocal in ...` 🤔

Reply:
I suspect this is because the inferred known-local bits aren't getting into the solution set, so they don't get applied to the `ParamDecl`s.

Reply:
Oh okey so that's going to be from the solution... great, I know how to proceed with those, thanks!